### ISSUE TITLE
Class  does not exist - PHP 7.3

### ISSUE THREAD

**Issue by ian-nisbet** [itsgoingd/clockwork#302]
Although the stack trace below is not very explanatory, Is anyone else using Laravel 5.5 getting a similar error after upgrading to PHP7.3? Removing `"itsgoingd/clockwork"`: "^3.0" from `composer.json` gets rid of the problem. 

Incidentally, upgrading from v2.4 to v3.2 of https://github.com/barryvdh/laravel-debugbar prevented the same "Class  does not exist" error message.

```
[2018-12-10 17:28:55] development.ERROR: Class  does not exist {"exception":"[object] (ReflectionException(code: -1): Class  does not exist at /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Container/Container.php:752)
[stacktrace]
#0 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Container/Container.php(752): ReflectionClass->__construct('\\x00clockwork\\\\supp...')
#1 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Container/Container.php(631): Illuminate\\Container\\Container->build('\\x00clockwork\\\\supp...')
#2 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Container/Container.php(586): Illuminate\\Container\\Container->resolve('\\x00clockwork\\\\supp...', Array)
#3 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(732): Illuminate\\Container\\Container->make('\\x00clockwork\\\\supp...', Array)
#4 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Container/Container.php(1195): Illuminate\\Foundation\\Application->make('\\x00clockwork\\\\supp...')
#5 /Users/iannisbet/Sites/rma2/vendor/itsgoingd/clockwork/Clockwork/Support/Laravel/ClockworkServiceProvider.php(152): Illuminate\\Container\\Container->offsetGet('\\x00clockwork\\\\supp...')
#6 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(586): Clockwork\\Support\\Laravel\\ClockworkServiceProvider->register()
#7 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php(75): Illuminate\\Foundation\\Application->register(Object(Clockwork\\Support\\Laravel\\ClockworkServiceProvider))
#8 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(561): Illuminate\\Foundation\\ProviderRepository->load(Array)
#9 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/RegisterProviders.php(17): Illuminate\\Foundation\\Application->registerConfiguredProviders()
#10 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(213): Illuminate\\Foundation\\Bootstrap\\RegisterProviders->bootstrap(Object(Illuminate\\Foundation\\Application))
#11 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(162): Illuminate\\Foundation\\Application->bootstrapWith(Array)
#12 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(146): Illuminate\\Foundation\\Http\\Kernel->bootstrap()
#13 /Users/iannisbet/Sites/rma2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
#14 /Users/iannisbet/Sites/rma2/public/index.php(67): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
#15 /Users/iannisbet/.composer/vendor/laravel/valet/server.php(151): require('/Users/iannisbe...')
#16 {main}
"}
```

**Comment by itsgoingd** [comment]
Looks like it might be this PHP 7.3 bug - https://bugs.php.net/bug.php?id=77275

I've run into similar issues multiple times after upgrading to 7.3. Seems pretty bad, hope 7.3.1 gets released soon, might need to disable opcache or wait with the upgrade till then.

**Comment by itsgoingd** [comment]
Fixed in PHP 7.3.2+.
